jobs/component_jobs.groovy

```diff
@@ -137,7 +137,8 @@ repos.each { Map repo ->
 
               export IMAGE_PREFIX=deisci VERSION="git-\${git_commit:0:7}"
               docker login -e="\$DOCKER_EMAIL" -u="\$DOCKER_USERNAME" -p="\$DOCKER_PASSWORD"
-              DEIS_REGISTRY='' make docker-build ${dockerPush}
+              # build once with "docker --pull --no-cache" to avoid stale layers
```

Comment:
We could--but then when we build once for Docker Hub and once for quay.io, the second build *won't* use cached layers and will `apt-get` and such all over again. That's slower and introduces the potential for them to disagree.

I think we want the two images to contain the same bits instead, so this is one way of doing that.